server/sonar-server/src/main/java/org/sonar/server/authentication/UserSessionInitializer.java

```diff
@@ -59,7 +59,7 @@
     "/setup/*",
     "/sessions/*",
     "/api/system/db_migration_status", "/api/system/status", "/api/system/migrate_db",
-    "/api/server/*",
+    "/api/server/index", "/api/server/setup", "/api/server/version",
```

Comment:
It's the list of URL that doesn't need to be authenticated.
When using "/api/server/*", it was badly excluding "/api/server_id". It's a limitation of this lib. 
In fact as described in the comment, these URLs should not be declared here but in the WS API.